.docker/docker-compose.dev.yml

```diff
@@ -74,6 +76,23 @@ services:
       options:
         max-size: "10m"
         max-file: "5"
+  video-transcoding-server:
+    image: ghcr.io/binarystudioacademy/streamlet-video-transoding-server-dev:latest
+    container_name: video-transcoding-server
+    volumes:
+      - segments:/app/backend/video-transcoding-server/playback
```

Comment:
It seems better to map the folder from the file system instance(local machine or instance in a cloud) to a file system in the docker container. In this case, it will be easier to migrate data to another location if necessary or find some problems in the work.